### ISSUE TITLE
Unable to paste when running headless on macOS

### ISSUE THREAD

**Issue by liamsharp** [TestFX/TestFX#590]
## Expected Behavior

To be able to call:

`robotContext().getTypeRobot().push(pasteKeyCombo);`

and have it past the text in.

## Actual Behavior

it doesn't work, and ends up entering the character v from the copy and paste short cut.

Demo project here:

https://github.com/liamsharp/testfx-pasting

Running (Headfull):

    mvn clean test 

works on Windows 10 & Ubuntu 16.04 and macOS 10.13.5.

Running (Headless)

    mvn clean test -PHeadless

works on Windows 10 & Ubuntu 16.04, but fails macOS 10.13.5.

## Specifications

  - Version: 10.13.5
  - Platform: macOS

**Comment by bob80333** [comment]
Have you tried using `KeyCombination.SHORTCUT_DOWN`?

From the KeyCombination [javadocs](https://docs.oracle.com/javase/8/javafx/api/javafx/scene/input/KeyCombination.html):

> The shortcut modifier is used to represent the modifier key which is used commonly in keyboard shortcuts on the host platform. This is for example control on Windows and meta (command key) on Mac. By using shortcut key modifier developers can create platform independent shortcuts. So the "Shortcut+C" key combination is handled internally as "Ctrl+C" on Windows and "Meta+C"

I modified the `getPaste()` method in your demo project to the following, and it started working correctly for me.

```
    private KeyCodeCombination getPaste()
    {
        return new KeyCodeCombination(KeyCode.V, KeyCombination.SHORTCUT_DOWN);
    }
```

**Comment by bob80333** [comment]
By checking SystemUtils.JAVA_AWT_HEADLESS, there is a workaround.

```private KeyCombination.Modifier getModifier()
    {
        return SystemUtils.IS_OS_MAC ? ("true".equals(SystemUtils.JAVA_AWT_HEADLESS) ? KeyCombination.SHORTCUT_DOWN : KeyCombination.META_DOWN) : KeyCombination.CONTROL_DOWN;
    }```

**Comment by liamsharp** [comment]
@bob80333 Excellent, that workaround does the trick.

**Comment by brcolow** [comment]
Should be fixed with #591
